core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -210,9 +211,14 @@ public void start() throws IOException {
   public void launchPollingThread(long mountId, long txId) {
     LOG.debug("launch polling thread for mount id {}, txId {}", mountId, txId);
     if (!mPollerMap.containsKey(mountId)) {
-      try (CloseableResource<UnderFileSystem> ufsClient =
-               mMountTable.getUfsClient(mountId).acquireUfsResource()) {
-        ufsClient.get().startActiveSyncPolling(txId);
+      UfsManager.UfsClient ufsClient = mMountTable.getUfsClient(mountId);
+      if (ufsClient == null) {
+        LOG.warn("Mount id {} does not exist", mountId);
```

Comment:
Is this expected to happen?

Reply:
nope, should i change it to log and throw an exception?

Reply:
Since this is launched internally I think it is fine to warn if the ramifications are not severe (in this case we would not respect active sync for a mount which doesnt exist?). If we crash it will just stop the thread but probably will not stop the system (ie. if we swallow the exception somewhere).